util/find-doc-nits

```diff
@@ -38,6 +39,7 @@ Find small errors (nits) in documentation.  Options:
     -u Count undocumented functions
     -h Print this help message
     -c List undocumented commands and options
+    -r List misorder EXAMPLES sections
```

Comment:
I would rather have `-o` to mean "List POD pages with misordered layout".  It's absolutely OK if you want to limit this effort to just check the order of EXAMPLES and RETURN VALUES for now, but we may want to expand that further, so we might as well imply that from the start.

Reply:
I also considered this so I agree with you

Reply:
Note: Please don't forget to change the commit message of b4d0c8d2ec0c13b22432aac5ef4860775a8d0aa7 accordingly (`-r` -> `-o`) when merging.

Reply:
Just a personal preference, but IMHO a brief explanation in the commit log would be helpful, because you get all information about the change without looking at the diffs.
```
Add -o option to find-doc-nits

-o: List POD pages with misordered layout
```

Reply:
@mspncp Sure! Will do what you suggested when I squash these commits ;-)